bin/ci/self-test/script.sh

```diff
@@ -44,3 +44,28 @@ fi
 if [[ "$ORCA_LIVE_TEST" ]]; then
   orca qa:automated-tests
 fi
+
+if [[ "$ORCA_ENABLE_NIGHTWATCH" == "TRUE" && "$ORCA_SUT_HAS_NIGHTWATCH_TESTS" && -d "$ORCA_YARN_DIR" &&  "$ORCA_SUT_NAME" == "drupal/example" ]]; then
```

Comment:
Niggle. 🙂
```suggestion
if [[ "$ORCA_ENABLE_NIGHTWATCH" == "TRUE" && "$ORCA_SUT_HAS_NIGHTWATCH_TESTS" && -d "$ORCA_YARN_DIR" && "$ORCA_SUT_NAME" == "drupal/example" ]]; then
```

Reply:
This is not required as we have removed the check `"$ORCA_SUT_NAME" == "drupal/example" `